core/src/martian/encoders.cljc

```diff
@@ -36,6 +36,9 @@
   ([] (default-encoders keyword))
   ([key-fn]
    (merge
+    #?(:bb
```

Comment:
For clj, we should support and prefer (in this order)
1. transit+msgpack
2. transit+json
3. edn
4. json

For bb and cljs it should just be 2, 3, 4 as they don't support msgpack.

So you can probably put msgpack for `:clj` only at the top, and merge that with a common list of 2, 3 and 4 to make it a bit more concise, i.e.

```clj
(merge
  #?(:clj {"application/transit+msgpack" ...})
  {"application/transit+json" ...
   "application/edn" ...
   "application/json" ...})
```

Reply:
This doesn't work for bb since bb will take the `:clj` branch if it's before the `:bb` branch.